wfd/public/peer.h

```diff
@@ -44,8 +44,9 @@ class Peer {
     /**
      * The implementation should send the RTSP data over the network
      * @param data data to be send
+     * @return false if still sending previous message
```

Comment:
Hmm, true. We talked with shalamov  a while ago and figured the problem doesn't actually happen  -- because we figured all our sends happen as a result of a received message or reply. But that of course is not true: there could be e.g. a keep-alive message and a user-initiated Pause right at the same time.

So: I'll have to accept all sends and store all current send-related event sources.